manifests/vanilla/vsphere-csi-driver.yaml

```diff
@@ -676,6 +676,7 @@ spec:
           args:
             - "--v=4"
             - "--csi-address=/csi/csi.sock"
+            - "--probe-timeout=4s"
```

Comment:
@jacen57 do you think this needs to updated in the documentation here - https://github.com/kubernetes-csi/livenessprobe?tab=readme-ov-file#command-line-options

Reply:
Can you update this for other liveness-probe container in the same file?
https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/ed4a34ac281bc907825f73466eab5a38a4374c6b/manifests/vanilla/vsphere-csi-driver.yaml#L532C17-L533